harness/config/commands.yml

```diff
@@ -112,6 +112,14 @@ command('console'):
     #!bash(workspace:/)|@
     passthru $COMPOSE_BIN exec -u "${CODE_OWNER}" console bash
 
+command('root <service>', 'root'):
```

Comment:
`ws root` is a bit odd I think, maybe `ws console-root`?

Reply:
a bit longer but closer convention is `ws console -u root`

Reply:
also the conventional support would get a 👍 from me anyway

Reply:
oh, <service> as well. hmm

Reply:
`ws shell [-u|--user <user>] <service>`?

Reply:
Updated as per your suggestion - as far as I understand the command parser, it needs the `=` in any option to consider it a value option instead of a bool,; if the `=` is left out, the value becomes an argument (which might still accidentally work depending on what your command reads). In this form it'll accept both short and long forms with or without `=` so we should be good here.